src/sbd/replacers/numbers.rs

```diff
@@ -36,20 +37,31 @@ impl fmt::Display for NumbersReplacer {
     }
 }
 
-impl Process for NumbersReplacer {
-    fn rules_vec(&self) -> Vec<Rule> {
-        self.rules.to_vec()
-    }
-}
-
 #[cfg(test)]
 mod tests {
     use super::*;
 
     #[test]
     fn default() {
         let numbers_replacer = NumbersReplacer::default();
-        println!("{:?}", numbers_replacer);
+        let default_rule = NumbersReplacer {
```

Comment:
i change my mind, i actually don't think we need this test tbh.... let's just remove it